repl/pom.xml

```diff
@@ -146,7 +144,7 @@
             </goals>
             <configuration>
               <sources>
-                <source>${extra.source.dir}</source>
+                <source>src/main/scala-${scala.binary.version}</source>
```

Comment:
Not strictly necessary right here, but I'm standardizing how we declare parallel scala source trees